py/tests/set.py

```diff
@@ -56,6 +56,15 @@
 d = a ^ b
 assert 1 in c
 
+doc="__repr__"
+a = {1, 2, 3}
+b = a.__repr__()
```

Comment:
what about this:
```suggestion
assert "{1, 2, 3}" == a.__repr__()
```

instead?

Reply:
@sbinet Set is an unordered data structure and I don't think comparing return value of __repr__ with string will be good test case since order of elements are not guaranteed to be same.

Reply:
Right :)